client/pages/cart.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+
+export default class Cart extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cartItems: []
+    };
+    this.cartItems = this.cartItems.bind(this);
+    this.subtotal = this.subtotal.bind(this);
+    this.cart = this.cart.bind(this);
+  }
+
+  componentDidMount() {
+    const token = window.localStorage.getItem('token');
+    if (token) {
+      fetch('api/products/cart/{this.props.cartId}', {
+        method: 'GET',
+        headers: {
+          'X-Access-Token': token
+        }
+      })
+        .then(res => res.json())
+        .then(cart => this.setState({ cartId: this.props.cartId, cartItems: cart }))
+        .catch(err => console.error(err));
+    }
+  }
+
+  cartItems() {
+    const items = this.state.cartItems;
+    const listItems = items.map((item, index) => {
+      return <ListGroup.Item key={index}>
+        <div className="d-flex border border-1 rounded py-2 cart-item">
+          <div className="col-6 text-center">
+            <img className="cart-img img-fluid" src={item.imageUrl} />
+          </div>
+          <div className="cart-details col-6 d-flex flex-column justify-content-center">
+            <h6 className="fw-light">{item.name} Snowboard</h6>
+            <h6 className="fw-light">Size: {item.size}</h6>
+            <h6 className="fw-light">Qty: <span className="fw-bold">1</span></h6>
+            <h6 className="fw-light">
+              Price: <span className="fw-bold">${item.price / 100}</span>
+            </h6>
+          </div>
+        </div>
+      </ListGroup.Item>;
+    });
+    return (<ListGroup className="col-md-8 mb-3">{listItems}</ListGroup>);
+  }
+
+  subtotal() {
+    let subtotal = 0;
+    this.state.cartItems.map(item => {
+      return (subtotal += item.price);
+    });
```

Comment:
If you have no need for the array created by `.map()`, it's best to use `.forEach()` instead.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach